ComposureWatch/src/Components/Header/HeaderListItemChevron.jsx

```diff
@@ -2,14 +2,16 @@ import ChevronSvg from "../SVG/ChevronSvg";
 
 const HeaderListItemChevron = ({ info }) => {
   return (
-    <li className="flex flex-row items-center">
+    <li className="flex flex-row items-center ">
```

Comment:
Dang added the extra space in there huh? 😄

Reply:
Have to keep you on your toes!